src/programs/ectrans-benchmark.F90

```diff
@@ -571,16 +569,16 @@ program ectrans_benchmark
 
 if (verbosity >= 0 .and. myproc == 1) then
   write(nout,'(" ")')
-  write(nout,'(a,i6,a,f9.2,a)') "ectrans_benchmark initialisation, on",nproc,&
+  write(nout,'(a,i0,a,f9.2,a)') "ectrans_benchmark initialisation, on",nproc,&
```

Comment:
Haven't seen `i0` before. Does that automatically scale the width of the printed integer?

Reply:
Correct